pkg/api/v1alpha2/edgeconnect/edgeconnect_types.go

```diff
@@ -37,35 +36,37 @@ type EdgeConnectSpec struct { //nolint:revive
 	// +kubebuilder:validation:Optional
 	Proxy *proxy.Spec `json:"proxy,omitempty"`
 
+	// ServiceAccountName that allows EdgeConnect to access the Kubernetes API
+	ServiceAccountName *string `json:"serviceAccountName,omitempty"`
+
+	// Enables automatic restarts of EdgeConnect pods in case a new version is available (the default value is: true)
+	AutoUpdate *bool `json:"autoUpdate"`
+
```

Comment:
@chrismuellner I update the EC CRD aswell, but it will have no new version in the next release, should I undo it, or is it fine? (and create a follow up to track it)

I can just revert [Remove kubebuilder:default from EdgeConnect](https://github.com/Dynatrace/dynatrace-operator/pull/4010/commits/f74846fbfb3c909d5dca5518466572defb1f888e), so it's not much work

Reply:
do we consider this a customer facing change? I think a customer shouldn't notice this change

Reply:
ok ok